services/db/migrations/050_multisig_alias_schemas.up.sql

```diff
@@ -0,0 +1,16 @@
+##
+## Multisig alias
+##
+create table multisig_aliases (
+    alias           varchar(50) not null,
```

Comment:
The size of our data structures are fixed, so maybe it is way faster / storage friendly to use this:
- txID: 49 chars (cb58 encoded 32 bytes) = char(49)
- alias / owner: 33 chars (cb58 encoded 20 bytes) = char(33)